### DISCUSSION TITLE
Human Gene expression DB

### DISCUSSION THREAD

**Discussion by Radwa100** [Open-Systems-Pharmacology/Forum#1779] [Questions & Problems]
Hi,
I downloaded ADME only  gene expression DB. My question is What make the human ADME  gene expression DB a very big number, roughly 54,000, even though the ADME gene is 300 gene only? where this number comes from? Thank you!

**Comment by HenrikCordes** [comment]
Dear @Radwa100 ,

thank you for your patinets. 
Can you share which release/version/name of the DB you download?  
Regarding the high number. Is this the number of different genes in the DB or related to the total number of expression entries?
 
Bear in mind that the database contains the expression information for a gene originating from multiple studies differentiating between tissue, gender, race/strain and age. Thus, for a single gene there will serveral entries in the database. 

Best
Henrik

**Reply by bhim1992** [reply]
Can you please explain for TPN only gene and DB gene

**Reply by HenrikCordes** [reply]
Dear @bhim1992 ,
you can find the diefinition of TPM here: https://www.rna-seqblog.com/rpkm-fpkm-and-tpm-clearly-explained/
